src/main/java/org/jabref/gui/texparser/ParseTexDialogViewModel.java

```diff
@@ -66,7 +66,7 @@ public ParseTexDialogViewModel(BibDatabaseContext databaseContext, DialogService
         this.taskExecutor = taskExecutor;
         this.preferencesService = preferencesService;
         this.fileMonitor = fileMonitor;
-        this.texDirectory = new SimpleStringProperty(databaseContext.getMetaData().getLaTexFileDirectory(preferencesService.getUser())
+        this.texDirectory = new SimpleStringProperty(databaseContext.getMetaData().getLatexFileDirectory(preferencesService.getUser())
```

Comment:
Why not also 'this.latexDirectory' and ParseLatexDialogViewModel.java?

Reply:
I will do that in a follow-up PR to keep things going.